src/engraving/style/style.h

```diff
@@ -98,7 +102,9 @@ class MStyle
     std::array<Millimetre, size_t(Sid::STYLES)> m_precomputedValues;
 
     void readVersion(String versionTag);
+    void readStylePreset(String presetTag);
     int m_version = 0;
+    String m_stylepreset = String();
```

Comment:
This variable should be the `AccessibilityStylePreset` enum rather than a string representation. You should convert to and from the string representation at the point where you save/load the score.